extensions/amp-story/1.0/amp-story.css

```diff
@@ -268,6 +268,12 @@ amp-story-grid-layer {
   bottom: 0 !important;
   left: 0 !important;
   z-index: 2 !important;
+  /** Make sure grid layer does not act as a click shield on elements underneath */
+  pointer-events: none !important;
+}
+
+amp-story-grid-layer > * {
```

Comment:
I wonder if this is the right selector; this will also mean that wrapper `<div>`'s that aren't visible will act as a click shield.  Do we actually need to do this, or can we just set it in JS when we find them?

Reply:
`pointer-events: auto` just makes elements behave as they would if `pointer-events` wasn't specified. This is just to override the `pointer-events: none` that we set above. 

I also tested this on my [demo](https://stamp-demo.firebaseapp.com/examples/amp-story/artezan/clickable-anchor.html) by placing a wrapper div on top of one of the links that triggers a tooltip, and it in fact doesn't act as a click shield, since I can still click the link underneath it.

Is this what you were asking?

Reply:
This is what I was asking, but rather than the clickable elements being inside of the container div, I mean the container div is on another layer on top of the clickable elements.  I would expect it to act as a click shield.  Put another way: why have anything accept click events except for the specific whitelist of selectors that we deem clickable?